managers/bosh-manager/DirectorService.js

```diff
@@ -0,0 +1,1112 @@
+      logger.error(`subnet ${this.networkName} definition not found among the applicable networks defintion : ${JSON.stringify(networks)}`);
+      throw new errors.UnprocessableEntity(`subnet ${this.networkName} definition not found`);
+    }
+    let manifestYml = _.template(this.template)(context);
+    if (!skipAddOns) {
+      const serviceManifest = yaml.safeLoad(manifestYml);
+      this.configureAddOnJobs(serviceManifest, context.spec);
+      manifestYml = yaml.safeDump(serviceManifest);
+    }
+    return manifestYml;
+  }
+
+  configureAddOnJobs(serviceManifest, context) {
+    const addOns = new Addons(context).getAll();
+    if (serviceManifest.addons) {
+      serviceManifest.addons = serviceManifest.addons.concat.apply(serviceManifest.addons, addOns);
+    } else {
+      serviceManifest.addons = addOns;
+    }
+  }
+
+  // findDeploymentTask(deploymentName) {
+  //   return this.director
+  //     .getTasks({
+  //       deployment: deploymentName
+  //     }, true)
+  //     .then(tasks => _
+  //       .chain(tasks)
+  //       .sortBy('id')
+  //       .find(task => /^create\s+deployment/.test(task.description))
+  //       .value()
+  //     );
+  // }
+
+  // getDeploymentInfo(deploymentName) {
+  //   const events = {};
+  //   const info = {};
+
+  //   function DeploymentDoesNotExist(err) {
+  //     return err.status === 404 && _.get(err, 'error.code') === 70000;
+  //   }
+
+  //   function addInfoEvent(event) {
+  //     if (!_.has(events, event.stage)) {
+  //       events[event.stage] = {
+  //         tags: event.tags,
+  //         total: event.total,
+  //       };
+  //     }
+  //     if (!_.has(events[event.stage], event.task)) {
+  //       events[event.stage][event.task] = {
+  //         index: event.index,
+  //         time: event.time,
+  //         status: event.state
+  //       };
+  //     } else {
+  //       events[event.stage][event.task].status = event.state;
+  //       let seconds = event.time - events[event.stage][event.task].time;
+  //       delete events[event.stage][event.task].time;
+  //       events[event.stage][event.task].duration = `${seconds} sec`;
+  //     }
+  //   }
+
+  //   return this
+  //     .findDeploymentTask(deploymentName)
+  //     .tap(task => _.assign(info, task))
+  //     .then(task => this.director.getTaskEvents(task.id))
+  //     .tap(events => _.each(events, addInfoEvent))
+  //     .return(_.set(info, 'events', events))
+  //     .catchReturn(DeploymentDoesNotExist, null);
+  // }
+
+  getApplicationAccessPortsOfService() {
+    let service = this.service.toJSON();
+    return _.get(service, 'application_access_ports');
+  }
+
+  buildIpRules() {
+    let applicationAccessPorts = this.getApplicationAccessPortsOfService();
+    return _.map(this.getNetwork(this.networkSegmentIndex), net => {
+      return {
+        protocol: 'tcp',
+        ips: net.static,
+        applicationAccessPorts: applicationAccessPorts
+      };
+    });
+  }
+
+  // getInfo() {
```

Comment:
remove commented out code.

Reply:
This methods are copied from old Instance/Manager files. They are still used with the old flow as part of dashboard show, which should eventually be removed. Kept it here so that we can uncomment it here and remove from old Instance/Manager files. Also we decided not to touch the  Instance/Manager files in the current PR. They also will be cleaned up.